src/Contants/Utils.tsx

```diff
@@ -397,4 +401,154 @@ export const updateQuickBlockUser = (updateduserProfile: any) => {
     .catch(function (e) {
       console.error('Error in updating user====', e);
     });
-};
\ No newline at end of file
+};
+
+export const sendMessage = async (
+  messageType: any,
+  content: any,
+  user: any,
+  setMessages: any,
+  dialog: any,
+  setNewMessage: any,
+  friend: any,
+  setSending: any,
+  minutes?: any,
+  seconds?: any,
+) => {
+  let newMsg: any = {};
+
+  if (messageType === 'text') {
+    if (content.trim() === '') {
+      return;
+    }
+

```

Comment:
Modify this if else conditions. Apply conditions on key values not overall Object